src/admin-views/settings/tabs/display/display-calendar.php

```diff
@@ -33,6 +34,9 @@
 	'default' => esc_html__( 'Default Page Template', 'the-events-calendar' ),
 ];
 
+// Allow integrations to modify template options.
+$template_options = apply_filters( 'tec_events_template_options', $template_options );
```

Comment:
```suggestion
$template_options = apply_filters( 'tec_events_template_options', $template_options );
```

Follow the filter naming convention as done for similar filter below: https://github.com/the-events-calendar/the-events-calendar/blob/a69b1238fce504ab10752cb4b55b7b78dcb9b9a5/src/admin-views/settings/tabs/display/display-calendar.php#L189

Also, add filter doc blocks.